changelogs/0.2.0-rc1.yaml

```diff
@@ -0,0 +1,34 @@
+date: August 31, 2022
+
+changes:
+- area: documentation
+  change: |
+    Added a quickstart guide for users to run and use Envoy Gateway.
+- area: api
+  change: |
+    Added the EnvoyGateway API type for configuring Envoy Gateway.
+    Added the EnvoyProxy API type for configuring managed Envoys.
+- area: ci
+  change: |
+    Added tooling to build, run, etc. Envoy Gateway.
+- area: providers
+  change: |
+    Added the Kubernetes provider.
+- area: xds
```

Comment:
nit: there are two separate sub components here - `xds-translator` and `xds-server`